Tests/SonarScanner.MSBuild.Tasks.IntegrationTests/E2ETests/E2EAnalysisTests.cs

```diff
@@ -931,12 +931,25 @@ public void AssertExpectedFileList(params string[] fileNames)
                 var actualFilesToAnalyze = ProjectInfo.AssertAnalysisResultExists("FilesToAnalyze");
                 actualFilesToAnalyze.Location.Should().Be(filesToAnalyzeFile.FullPath);
 
+                AssertFileIsUtf8Bom(filesToAnalyzeFile.FullPath);
+
                 if (ProjectConfig != null)
                 {
                     ProjectConfig.FilesToAnalyzePath.Should().Be(filesToAnalyzeFile.FullPath);
                 }
             }
 
+            private void AssertFileIsUtf8Bom(string filePath)
+            {
+                using (var filesToAnalyzeStream = File.Open(filePath, FileMode.Open, FileAccess.Read, FileShare.Read))
```

Comment:
This should do (almost) the same thing on a shorter line, the `ReadWrite` access doesn't hurt here.
```suggestion
                using (var filesToAnalyzeStream = File.Open(filePath, FileMode.Open))
```